client/post-editor/editor-ground-control/index.jsx

```diff
@@ -299,37 +353,7 @@ export class EditorGroundControl extends PureComponent {
 						</span>
 					</div>
 				) }
-				{ ( isSaveAvailable || shouldShowStatusLabel ) && (
-					<div className="editor-ground-control__status">
-						{ isSaveAvailable && (
-							<button
-								className="editor-ground-control__save button is-link"
-								onClick={ this.onSaveButtonClick }
-								tabIndex={ 3 }
-							>
-								{ translate( 'Save' ) }
-							</button>
-						) }
-						{ ! isSaveAvailable &&
-						shouldShowStatusLabel && (
-							<span
-								className="editor-ground-control__save-status"
-								data-e2e-status={ isSaving ? 'Saving…' : 'Saved' }
-							>
-								{ isSaving ? translate( 'Saving…' ) : translate( 'Saved' ) }
-							</span>
-						) }
-					</div>
-				) }
-				{ hasRevisions && (
-					<HistoryButton
-						selectRevision={ this.props.selectRevision }
-						setNestedSidebar={ this.props.setNestedSidebar }
-						toggleSidebar={ this.props.toggleSidebar }
-						isSidebarOpened={ this.props.isSidebarOpened }
-						nestedSidebar={ this.props.nestedSidebar }
-					/>
-				) }
+				{ this.renderGroundControlQuickSaveButtons() }
```

Comment:
How do you feel about pulling this in to a component file at this point?
It could even make some sense to keep both the left and right action buttons together in one component 🤷‍♂️ , but maybe better to have them separate.

Reply:
@spen I actually had this pulled out into its own component, but it made this tougher to review & we'd need to refactor a bit to keep all the `tabIndex` declarations in a sane spot (i.e. not spread out inside child components).  Decided to punt it to a follow up PR.